dev/io.openliberty.microprofile.openapi.2.0.internal_fat/fat/src/io/openliberty/microprofile/openapi20/fat/deployments/MergeServerXMLTest.java

```diff
@@ -109,6 +111,36 @@ public void testInvalidServerXML() throws Exception {
 
     }
 
+    @Test
+    @Mode(TestMode.FULL)
+    public void testInvalidServerXMLEmptyNames() throws Exception {
+        server.setMarkToEndOfLog();
```

Comment:
These tests look good, but I think they'll conflict with #29957

Reply:
Sweet, once 29957 goes in I'll rebase.